webpack-tools.config.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env node */
```

Comment:
We may not need a webpack config for the tools and rely directly on our tsconfig.json instead.